css/css-overflow/line-clamp/webkit-line-clamp-044.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>CSS Overflow: -webkit-line-clamp creates an IFC</title>
+<link rel="author" title="Andreu Botella" href="mailto:abotella@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-overflow-4/#line-clamp">
+<link rel="match" href="reference/webkit-line-clamp-044-ref.html">
+<meta name="assert" content="-webkit-line-clamp always creates an independent formatting context, even without overflow: hidden, which causes the height to grow to the clearance of floats.">
+<style>
+  .clamp {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 5;
+    /* no overflow: hidden */
+    background-color: yellow;
+    padding: 0.5em;
+    font-size: 16px / 32px serif;
+    white-space: pre;
+  }
+  .float {
+    float: left;
+    background-color: orange;
+    padding: 0.5em;
+    height: 100px;
+    width: 100px;
+  }
```

Comment:
For the same reason as discussed for floats-008, this test works with `continue: collapse` but not with `continue: discard`. If you add  to the float, then I think it does work.
```suggestion
    break-inside: avoid;`
  }
```

Reply:
Similarly to above, there shouldn't be any unforced break in the float, so it should work with `continue: discard` as well.

Reply:
Yep, you're right. Sorry about that.